### DISCUSSION TITLE
Adding this code onto another code

### DISCUSSION THREAD

**Discussion by TurboTronix** [sle118/squeezelite-esp32#336] [General]
I started looking into this project: https://www.home-assistant.io/voice_control/thirteen-usd-voice-remote/
Basically it's a M5Stack ATOM Echo (which I believe is built on an ESP32) that controls HomeAssistant via voice (it has a mic and a speaker builtin). I was wondering if I can add to that project the code the ability to use the device as a spotify speaker. To do so I'll need to add the "squeeselite" part to it. I'm new to all of this but wondered if this is even doable.

**Comment by sle118** [comment]
"anything is doable" with enough time and resources. If your goal is to integrate Spotify, then start with cspot, which is the component that provides this feature. It is not for the faint of heart and definitely not a beginner project, but if you are patient, then why not?
